doc/help/CHECK.md

```diff
@@ -0,0 +1,83 @@
+kdb-check(1) -- Perform internal checks
+=======================================
+
+## DESCRIPTION
+This command is used to perform checks on the key database or an elektra plugin.  
+
+## USAGE
+`kdb check <plugin>`  
+
+Where `plugin` is the plugin that a user wants to check.  
+Use `-c` to pass options to that plugin.  
+If no `plugin` argument is provided a check will be performed on the key database itself.  
+The user can also use this tool to perform write tests by passing the `-f` option (please note that this can result in configuration files being changed!).  
+Special values are [returned](#RETURNED VALUES) to represent the outcome of a check.  
+
+## RETURNED VALUES
+Their are two different types of checks, a check on a plugin (by specifying the name of a plugin as an argument) or a check on the key database itself.  
+
+The outcome of a check on the key database is returned as an 8-bit pattern.  
+Each bit represents a specific outcome as described below:  
+
+ * 0:
+   No errors (no output)  
+
+ * Bit 1: 
+   Warning on open.  
+
+ * Bit 2:
+   Error on open.  
+
+ * Bit 3:
+   Warning on get.  
+
+ * Bit 4:
+   Error on get.  
+
+ * Bit 5:
+   Warning on set. (only checked when `-f` is used)  
+
+ * Bit 6:
+   Error on set (only checked when `-f` is used)  
+
+ * Bit 7:
+   Warning on close.  
+
+ * Bit 8:
+   Error on close.  
+
+So if the following bit pattern was returned: `010100000`  
```

Comment:
Any updates on this? Sorry I disappeared a bit yesterday and today, I had an interview and coding test to take care of :/ I will work on fixing this and a command or two tomorrow though.